tests/test_playbook_runs.py

```diff
@@ -121,13 +134,18 @@ def wrapper(*args, **kwargs):
 #   test_* methods.
 for group_name, group_tests in get_test_groups().items():
     _tests = {}
+
     for test_config in group_tests:
         test_name = test_config["name"].replace("-", "_")
         test_path = test_config["path"]
 
         @pytest.mark.skipif(
```

Comment:
Can't we use the skipif decorator more then once? 
If not might be better to wrap up the skip conditions in a function. For example: `@pytest.mark.skipif(**get_skip_conditions(group_name, test_name))`

Reply:
It did not work for me with multiple skipif, and I believe the order might matter, which is not good for our purposes. I'll add the function.